test/tests/multiapps/picard_catch_up_keep_solution/tests

```diff
@@ -0,0 +1,12 @@
+[Tests]
+  [./test]
+    type = 'Exodiff'
+    input = 'master.i'
+    exodiff = 'master_out_sub0.e master_out.e'
+#    max_parallel = 1 # This is here because we're counting residual evaluations
+#    max_threads = 1 # NanAtCountKernel changes behavior with threads
+#    recover = false
+#    petsc_version = '>=3.6.1'
```

Comment:
Why all the commented out parameters? Did you copy/paste this from a similar test? Do we want to commit it this way?

Reply:
Copy pasta - removed